File: 1019-squares-of-a-sorted-array/squares-of-a-sorted-array.py
class Solution:
    def sortedSquares(self, nums: List[int]) -> List[int]:
        # two pointer method
        l = 0;
        r = len(nums)-1;
        res = [0]* len(nums);
        while l<= r:
            leftNum = abs(nums[l]);
            rightNum = abs(nums[r]);
            if leftNum < rightNum:
                res[r - l] = rightNum *rightNum;
                r -=1;
            else:
              res[r-l] = leftNum * leftNum;
              l+=1;
        return res;

File: 1019-squares-of-a-sorted-array/squares-of-a-sorted-array.js
/**
 * @param {number[]} nums
 * @return {number[]}
 */
var sortedSquares = function(nums) {
    let arr = []
    for(let i = 0;i< nums.length; i++){
        arr.push(nums[i]* nums[i])
    }
    return arr.sort((a,b)=>a-b)
};